### PR TITLE
AG-8564 - Fix LineSeries and AreaSeries short-circuit

### PR DESCRIPTION
**Pull request by alantreadway**
Fixes short-circuiting of axes animation when line/area series rendering are not animatable.

### CODE REVIEW

File: packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts [hunk 1]
```diff
@@ -707,6 +701,7 @@ export class AreaSeries extends CartesianSeries<
 
         const fns = prepareAreaPathAnimation(newData, oldData, this.processedData?.reduced?.diff);
         if (fns === undefined) {
+            animationManager.skipCurrentBatch();
```
Comment (lsjroberts) [inline packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts:704]: Would this present issues when using on combined series types charts?

File: packages/ag-charts-website/src/content/docs/line-series-test/_examples/category-animation/main.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import { AgChartOptions, AgEnterpriseCharts, time } from 'ag-charts-enterprise';
 
+window.agChartsDebug = 'animation';
```
Comment (lsjroberts) [inline packages/ag-charts-website/src/content/docs/line-series-test/_examples/category-animation/main.ts]: intentional?

**Review by lsjroberts — APPROVED** [review]

File: packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts (hunk 1 above)
Comment (alantreadway) [reply packages/ag-charts-community/src/chart/series/cartesian/areaSeries.ts:704]: So it was always the intent that if one series didn't animate, that the others shouldn't either - so I believe this is desirable and consistent with that idea?

File: packages/ag-charts-website/src/content/docs/line-series-test/_examples/category-animation/main.ts (hunk 2 above)
Comment (alantreadway) [reply packages/ag-charts-website/src/content/docs/line-series-test/_examples/category-animation/main.ts]: Good spot, fixed

**Review by lsjroberts — APPROVED** [review]

**Pull request merged**